### ISSUE TITLE
NotifyPlugin: `aria-label` is ignored in `actions`

### ISSUE THREAD

**Issue by Anhil-111** [quasarframework/quasar#7477]
**Describe the bug**
When you create a notification via `this.$q.notify`, you can't set `aria-label` for a custom action.

**Codepen/jsFiddle/Codesandbox (required)**
https://jsfiddle.net/ms0kqt92/1/

Notifications don't seem to work on jsFiddle for me but I couldn't figure out how to fork on Codepen. Apologies.

**To Reproduce**
1. Go to jsFiddle;
2. Click on 'Notify' (if nothing happens, copy-paste JS code to https://codepen.quasar.dev and repeat);
3. Right-click on the notification's button, choose 'Inspect Element';
4. The button doesn't have `aria-label` set.

**Expected behavior**
I expect the button to have `aria-label`.

**Screenshots**
\-

**Platform (please complete the following information):**
OS: macOS 10.15.6
Node: 10.21.0
NPM: -
Yarn: 1.22.4
Browsers: Chrome 84.0.4147.89
iOS: -
Android: -
Electron: - 

**Additional context**
\-

**Comment by rstoenescu** [comment]
"attrs" will be available in Quasar v1.13.0.

**Comment by Anhil-111** [comment]
Thanks!
